### DISCUSSION TITLE
Why is the disk space occupied by my WeedFS increasing by approximately many times every day

### DISCUSSION THREAD

**Discussion by BenBenBenny** [seaweedfs/seaweedfs#5101] [Q&A]
On one day, approximately 11,000 images were uploaded, totaling around 3-4 gigabytes in size. However, the actual disk space occupied by WeedFS is much larger than this. Details are shown in the following figure.
os：
Windows server 2012
before：
![image](https://github.com/seaweedfs/seaweedfs/assets/20658585/c053b208-030a-4cb8-9582-4512f1cf49e8)
![image](https://github.com/seaweedfs/seaweedfs/assets/20658585/2570e0fb-c030-4df5-95cc-a0f39a8d1f56)
after：
![image](https://github.com/seaweedfs/seaweedfs/assets/20658585/06552ff4-0ccb-49a7-aad5-c9b2ff07be20)
![image](https://github.com/seaweedfs/seaweedfs/assets/20658585/2b6a86a6-3875-46d8-bda0-98092b139ee8)

the command ：
weed.exe  server -ip=127.0.0.1 -filer -filer.port=25925 -volume.port=9081 -volume.max=150    -dir=".\data"  -master.defaultReplication=000

**Comment by SmsS4** [comment]
If you delete files they occupy space till compaction
Also you can use `weed export` to list files

**Reply by BenBenBenny** [reply]
in our system, we rarely delete files.
We tested manual compaction using volume.vacuum in version 2.53  locally , and observed that the files became inaccessible. We are unsure of what happended.

about version 2.53，if the disk size is 3T, the volumn size 2.4T, the compaction can auto start ?

**Reply by ruslan-mogilevskiy** [reply]
@BenBenBenny , I saw somewhere here a suggestion to have a couple of your volume sizes of free space on the volume so the compaction could happen.
